### PR TITLE
Make MicroShift tests to be required for merge

### PR DESCRIPTION
**Pull request by oglok**
Signed-off-by: Ricardo Noriega <rnoriega@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift/microshift/openshift-microshift-main.yaml [hunk 1]
```diff
@@ -111,73 +111,73 @@ tests:
...
 - as: e2e-openshift-conformance-sig-storage
-  optional: true
+  optional: false
   run_if_changed: ^(cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
   steps:
     cluster_profile: gcp
     workflow: openshift-microshift-e2e-openshift-conformance-sig-storage
 - as: e2e-reboot
-  optional: true
+  optional: false
```
Comment (mangelajo) [inline ci-operator/config/openshift/microshift/openshift-microshift-main.yaml:180]:
```suggestion
  optional: true
```

Let's make this one optional for now until we figure out why is it always failing

**Review by sallyom — APPROVED** [review]
lgtm

**Pull request merged**